tasks/pipeline.py

```diff
@@ -174,6 +211,19 @@ def run(
     if here:
         git_ref = ctx.run("git rev-parse --abbrev-ref HEAD", hide=True).stdout.strip()
 
+    pipelines = get_running_pipelines_on_same_ref(gitlab, project_name, git_ref)
+
+    if len(pipelines) != 0:
+        print(
+            (
+                "There are already {} pipeline(s) running on the target git ref.\n"
+                "For each of them, you'll be asked whether you want to cancel them or not.\n"
+                "If you don't need these pipelines, please cancel them to save CI resources.\n"
+                "They are ordered from the newest one to the oldest one.\n"
+            ).format(len(pipelines))
```

Comment:
You can do
```suggestion
            "There are already {} pipeline(s) running on the target git ref.".format(len(pipelines)),
            "For each of them, you'll be asked whether you want to cancel them or not.",
            "If you don't need these pipelines, please cancel them to save CI resources.",
            "They are ordered from the newest one to the oldest one.",
            sep="\n")
```
which looks better IMO